server/routes/viewingIncidents/viewIncidents.ts

```diff
@@ -15,34 +16,42 @@ export default class ViewIncidentsRoutes {
   viewIncident: RequestHandler = async (req, res) => {
     const { incidentId } = req.params
     const { tab } = req.query
+    const yourReport = req.query['your-report']
     const { isReviewer, isCoordinator, username } = res.locals.user
-    const report = await this.reportService.getReport(req.user.username, parseInt(incidentId, 10))
+    const report = await this.reviewService.getReport(parseInt(incidentId, 10))
     const reportEdits = await this.reportService.getReportEdits(parseInt(incidentId, 10))
```

Comment:
reviewService gets reports irrespective of the user accessing them whereas reportService locked it down to currently logged in user